### DISCUSSION TITLE
Exceeded timeout of 300000ms while waiting for Service resource watchtower-kafka-external-bootstrap in namespace watchtower-kafka to be addressable

### DISCUSSION THREAD

**Discussion by bj9306** [strimzi/strimzi-kafka-operator#5058] [General]
Team,
I am getting below error, when installing kafka operator. I have attached yaml file of deployment config.

[kafka-persistent_tls.txt](https://github.com/strimzi/strimzi-kafka-operator/files/6573194/kafka-persistent_tls.txt)

Exceeded timeout of 300000ms while waiting for Service resource watchtower-kafka-external-bootstrap in namespace watchtower-kafka to be addressable

[kafka-k8s-components.txt](https://github.com/strimzi/strimzi-kafka-operator/files/6573201/kafka-k8s-components.txt)

Describe output of kafka operator.

[kubectl_describe_k.txt](https://github.com/strimzi/strimzi-kafka-operator/files/6573304/kubectl_describe_k.txt)

**Comment by scholzj** [comment]
You use a load balancer listener. So this most likely means that the load balancer is not being created by your environment:
```
watchtower-kafka-external-bootstrap   LoadBalancer   172.66.93.73     <pending>     9095:32105/TCP               7m24s
```
See the `<pending>` here.

So you will need to figure out why or use something else than load balancers.
